power-platform/admin/environment-groups.md

```diff
@@ -58,11 +58,11 @@ Enabling _default environment routing_ might give you more environments to manag
 
 ## Configure the rules for your environment group
 
-After you create the environment group, Power Platform tenant administrators can immediately add environments or configure the group's rules. Currently, there are six available rules, which are the same capabilities offered when upgrading an _individual_ environment to a [Managed Environment](managed-environment-overview.md).
+After you create the environment group, Power Platform tenant administrators can immediately add managed environments or configure the group's rules. Currently, there are six available rules, which are the same capabilities found in [Managed Environment](managed-environment-overview.md). 
 
 ### The sharing limits rule
 
-You can also configure the _sharing limits_ rule. Since the environment group is intended for personal productivity, makers are restricted from sharing their canvas apps with other users. This helps ensure that each environment in the group remains a private space for individual work.
+You can also configure the _sharing limits_ rule. Since the environment group is intended for managed personal productivity, makers are restricted from sharing their canvas apps with other users. This helps ensure that each environment in the group remains a private space for individual work.
```

Comment:
The prefix "managed" is not appropriate in this sentence. I suggest we keep "personal productivity" all the while including the other changes proposed in this PR.